### ISSUE TITLE
convert fLoc/rois to a dataset and add as a submodule 

### ISSUE THREAD

**Issue by MarieStLaurent** [courtois-neuromod/cneuromod-things#32]
- [x] From local repo copy on beluga, convert `fLoc/rois` directory and its entire content into a cneuromod datalad dataset called `floc.rois`

Local path on beluga: `projects/rrg-pbellec/mstlaure/cneuromod-things/fLoc/rois`

Description: fLoc vision localizer derivatives and documentation, including contrast maps and binary ROI masks.

**UPDATE: keep files for version control**; (do not) remove files downloaded from the Kanwisher group's website from the dataset; keep them to garantee version control?).  `fLoc/rois/standard_masks/kanwisher_parcels/cvs/cvs_*_parcels` folders from final dset (present in local repo on beluga). 

- [x] push data to s3

- [x] Then, install `floc.rois` as a submodule called `rois` directly under `cneuromod-things/fLoc` to replace the placeholder folder (not a repo). Update .gitignore file to track the submodule's content.

**Comment by bpinsard** [comment]
@MarieStLaurent  is floc.rois the right name, as it also contains GLM maps too? One could be interested in the GLM maps rather than ROIs.

floc.??? floc.glm_rois  floc.analysis floc.

**Comment by MarieStLaurent** [comment]
true, but I think it's more about the ROIs, the GLMs are really just a mean to an end. I think I prefer to keep it as floc.rois since it's what most users will be looking for
